README.md

```diff
@@ -1,70 +1,9 @@
-  * Enable annotation processing (for Lombok): `File` -> `Settings` -> `Build`, `Execution`, `Deployment` -> `Compiler` -> `Annotation Processors` -> `Enable Annotation Processing`
-  * Set Project SDK to JDK 11: `Project Structure` -> `Project`
-
-### Set up project
-
-From the main project folder, run the following commands:
-
-```
-git config --local include.path ../.gitconfig
-ln -s ../../scripts/git-hooks/machete-pre-rebase .git/hooks/machete-pre-rebase
-ln -s ../../scripts/git-hooks/machete-status-branch .git/hooks/machete-status-branch
-ln -s ../../scripts/git-hooks/post-commit .git/hooks/post-commit
-ln -s ../../scripts/run-pre-build-checks .git/hooks/pre-commit
-```
----
-Consider increasing maximum heap size for the IDE (the default value is 2048 MB). Go to `Help` -> `Change Memory Settings` for this.
-
-### Run/Debug
-
-To run an instance of IDE with Git Machete IntelliJ Plugin execute `:runIde` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `runIde` or `./gradlew runIde`).
+![](src/main/resources/META-INF/pluginIcon.svg)
 
-#### Logging
-
-Logging of this plugin has several categories:
-* `backend`
-* `backendRoot`
-* `binding`
-* `branchLayout`
-* `frontendActions`
-* `frontendUiRoot`
-* `frontendUiTable`
-* `gitCore`
-
-By default, IntelliJ logs everything with level `INFO` and above into `idea.log` file.
-The exact location depends on IntelliJ version and OS, check `Help` -> `Show Log in Files` to find out.
-
-To enable logging in `DEBUG` level, add selected categories to list in `Help` -> `Diagnostic Tools` -> `Debug Log Settings`.
-The standard practice in Java logging is to use fully-qualified name of each class as category;
-in our case, however, we're only ever using the categories provided above.
-
-### Generate plugin zip
-
-To generate a plugin archive run `:buildPlugin` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `buildPlugin` or `./gradlew buildPlugin`).
-The resulting file will be available under `build/distributions/`.
-
-### Install snapshot build of the plugin
-
-Download the zip file from the artifacts of the given build in [CircleCI](https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin).
-Go to `File` -> `Settings` -> `Plugins` -> `(gear icon)` -> `Install Plugin from Disk...`, select the zip and restart IDE.
-
-### Rebuild the CI base image
+This is a port of [git-machete](https://github.com/VirtusLab/git-machete) into an IntelliJ plugin.
 
-To push the rebuilt image, you need write access to [`gitmachete` organization on Docker Hub](https://hub.docker.com/orgs/gitmachete).
+This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
 
-```
-DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain docker build -t gitmachete/intellij-plugin-ci .
-docker push gitmachete/intellij-plugin-ci
-```
+For details on contributing, see the [guideline](CONTRIBUTING.md).
```

Comment:
I think that moving info about building into `CONTRIBUTING.md` may not be a good idea (can be misleading) - someone may want to just build our plugin from source but not contributing.
So maybe split build related information to another file (`BUILDING.md`?) or just leave it as it was - in this file.

Reply:
Right, good point... I actually have quite a few OSS projects checked out locally:

![image](https://user-images.githubusercontent.com/3383210/81830566-829b1780-953c-11ea-898a-ac865b3f8d60.png)

some of which I might want to build myself, without ever contributing.

With that in mind, I revamped README and CONTRIBUTING;  README now contains all the things that might in my assessment be ever needed to someone who builds the project locally but doesn't contribute.